source/common/json/json_loader.cc

```diff
@@ -154,6 +155,33 @@ class ObjectImplBase : public Object {
 
   bool hasObject(const std::string& name) const override { return value_.HasMember(name.c_str()); }
 
+  void validateSchema(const std::string& schema) const override {
+    rapidjson::Document document;
+    if (document.Parse<0>(schema.c_str()).HasParseError()) {
+      throw std::invalid_argument(fmt::format("invalid schema \n Effor(offset {}) : {}\n",
+                                              document.GetErrorOffset(),
+                                              GetParseError_En(document.GetParseError())));
+    }
+
+    rapidjson::SchemaDocument schema_document(document);
```

Comment:
Don't you need to check some type of error here and throw if the schema itself is not valid?